webui/react/src/hooks/useModal/UserSettings/useModalCreateUser.tsx

```diff
@@ -179,14 +127,6 @@ const ModalForm: React.FC<Props> = ({ form, user, groups, viewOnly, roles }) =>
           </Select>
         </Form.Item>
       )}
-      {!!user && rbacEnabled && canGetPermissions && (
-        <Table
-          columns={permissionTableColumn}
-          dataSource={permissions}
-          pagination={{ hideOnSinglePage: true, size: 'small' }}
-          rowKey="name"
-        />
-      )}
```

Comment:
Remove legacy table to display each permissions (this never showed up because canGetPermissions always false, explained in detail in usePermissions)